src/parser.pegjs

```diff
@@ -39,6 +39,9 @@
     "!": "semantic_not"
   };
 }}
+{
+  var reservedWords = new Set(options.reservedWords);
```

Comment:
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#browser_compatibility), and [our compatibility list](https://github.com/peggyjs/peggy#compatibility) we are losing Internet Explorer. Neither TypeScript, nor rollup, nor terser does not polyfill `Set`, so we are not ready to use `Set` here. I've even checked TypeScript in the ES3 mode. Nothing.

So I change it to the ordinary array. That should not be a big problem, because linear search in the list won't be much worse because that list hardly ever will big enough to see `Set` benefits